Xwt.Mac/NSStringAttributeKey.cs

```diff
@@ -0,0 +1,43 @@
+﻿using MonoMac.Foundation;
+
+namespace Xwt.Mac
+{
+	public static class NSStringAttributeKey
+	{
+		public static NSString ForegroundColor
+		{
+			get { return NSAttributedString.ForegroundColorAttributeName; }
```

Comment:
@Therzok  Could you elaborate on how exactly this would break Xamarin.Mac? Because the code is only in Xwt.Mac, not in Xwt.XamMac.